processor/src/chiplets/memory/tests.rs

```diff
@@ -557,20 +558,18 @@ fn build_trace_row(
     row[V_COL_RANGE.start + 2] = word_values[2];
     row[V_COL_RANGE.start + 3] = word_values[3];
 
-    if prev_row != [ZERO; MEMORY_TRACE_WIDTH] {
-        let delta = if row[CTX_COL_IDX] != prev_row[CTX_COL_IDX] {
-            row[CTX_COL_IDX] - prev_row[CTX_COL_IDX]
-        } else if row[WORD_COL_IDX] != prev_row[WORD_COL_IDX] {
-            row[WORD_COL_IDX] - prev_row[WORD_COL_IDX]
-        } else {
-            row[CLK_COL_IDX] - prev_row[CLK_COL_IDX] - ONE
-        };
-
-        let (hi, lo) = super::split_element_u32_into_u16(delta);
-        row[D0_COL_IDX] = lo;
-        row[D1_COL_IDX] = hi;
-        row[D_INV_COL_IDX] = delta.inv();
-    }
+    let delta = if row[CTX_COL_IDX] != prev_row[CTX_COL_IDX] {
+        row[CTX_COL_IDX] - prev_row[CTX_COL_IDX]
+    } else if row[WORD_COL_IDX] != prev_row[WORD_COL_IDX] {
+        row[WORD_COL_IDX] - prev_row[WORD_COL_IDX]
+    } else {
+        row[CLK_COL_IDX] - prev_row[CLK_COL_IDX]
+    };
```

Comment:
Questions: why do we not need `if prev_row != [ZERO; MEMORY_TRACE_WIDTH]` check any more?